src/clipboard.ts

```diff
@@ -115,7 +115,8 @@ const getSystemClipboard = async (
   try {
     const text = event
       ? event.clipboardData?.getData("text/plain").trim()
-      : probablySupportsClipboardReadText &&
+      : "clipboard" in navigator &&
+        "readText" in navigator.clipboard &&
         (await navigator.clipboard.readText());
```

Comment:
I’ve not added a comment here, but I have in #5786. In this case to create an image for testing i am pasting an image, since the clipboard property is not present in jest, i had to create a fake clipboard object and and evaluate its presence at runtime.

Reply:
My initial intention was to test the flip pixel by pixel of the canvas before and after. 

For now tests are only covering the correctness of the bounding box of the objects before and after. It will be cool to test that the actual flipping occurred correctly graphically, but I wasn't able to do it in jest. 
Other improvements of the tests include:
- clearing local storage before each test.
- take into account objects that have curve inside/outside min/max points bounds.
- differentiate between 2 point and multipoint linear elements.

Reply:
👍 cool. So I'll revert the clipboard change since it's not really doing anything.